obal/data/roles/setup_sources/tasks/specfile.yml

```diff
@@ -1,16 +1,10 @@
 ---
-- name: get spec file
-  find:
-    pattern: "*.spec"
-    path: "{{ package_dir }}"
-  register: setup_sources_spec_file
-
 - name: 'Set spec file path'
   set_fact:
-    setup_sources_spec_file_path: "{{ setup_sources_spec_file.files[0].path }}"
+    spec_file_path: "{{ spec_file.files[0].path }}"
```

Comment:
This is only used in one place. Perhaps just use `spec_file.files[0].path` in the `spectool` command below?

Reply:
I decided to move `spec_file_path` assignment into the `spec_file` role.